hikari/applications.py

```diff
@@ -89,6 +89,12 @@ class ApplicationFlags(enums.Flag):
     EMBEDDED = 1 << 17
     """Denotes that the application has functionality that's specially embedded in Discord's client."""
 
+    GATEWAY_MESSAGE_CONTENT = 1 << 18
```

Comment:
Not sure whether these should be called GATEWAY_MESSAGE_CONTENT* or just MESSAGE_CONTENT* rn

Reply:
I suspect it should be `MESSAGE_CONTENT*`, as it wouldn't make sense for them to not send the information in the gateway but yes in the REST API.